integration-cli/events_utils.go

```diff
@@ -130,7 +130,7 @@ func matchEventLine(id, eventType string, actions map[string]chan bool) eventMat
 
 		if matchIDAndEventType(matches, id, eventType) {
 			if ch, ok := actions[matches["action"]]; ok {
-				close(ch)
+				ch <- true
```

Comment:
we don't really care if the channel is closed or not, we only want to notify the main goroutine.